sources/packages/web/src/constants/message-constants.ts

```diff
@@ -9,3 +9,8 @@ export const PENDING_SIN_MESSAGE =
  */
 export const INVALID_SIN_MESSAGE =
   "StudentAid BC has found an issue with your account information. Please ensure your account information matches CRA and log back in. If you have further issues please contact us.";
+
+/**
+ * toast display time
+ */
+export const TOAST_ERROR_DISPLAY_TIME = 15000;
```

Comment:
Can we move it inside the useToastMessage.ts and maybe called it EXTENDED_MESSAGE_DISPLAY_TIME